mythril/laser/ethereum/instructions.py

```diff
@@ -1842,6 +1842,14 @@ def call_(self, global_state: GlobalState) -> List[GlobalState]:
             callee_address, callee_account, call_data, value, gas, memory_out_offset, memory_out_size = get_call_parameters(
                 global_state, self.dynamic_loader, True
             )
+
+            if callee_account is not None and callee_account.code.bytecode == "":
+                log.debug("The call is related to ether transfer between accounts")
```

Comment:
without this, it seems to cause some weird index errors(due to the way cfg is handled) if this isn't placed here. It's a bit hard to place it on the transaction handling area so this might be a quicker way to fix that.

Reply:
With my pr #1025 we might not have this problem anymore